Orchestrator/v0.2/nlr_versions.json

```diff
@@ -2,7 +2,7 @@
   "version": "0.2",
   "default": {
     "en_intent": "pretrained.20200924.microsoft.dte.00.06.en.onnx",
-    "multi_intent": "pretrained.20201210.microsoft.dte.00.12.unicoder_multilingual.onnx"
+    "multilingual_intent": "pretrained.20201210.microsoft.dte.00.12.unicoder_multilingual.onnx"
```

Comment:
perhaps, we could do these instead if you are concerned about multi_intent being confused as multiple intent.

intent_en: ...
intent_multi: ...
entity_en: ...
entity_multi: ...

Reply:
Yeah, I was worried about the confusion. "multi" is just too vague.